kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,62 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+
+    uint8_t readRtcRegister(uint8_t rtcRegister)
+    {
+        CPU::PortWrite8(CMOS_ADDRESS_REGISTER, rtcRegister);
+        return CPU::PortRead8(CMOS_DATA_REGISTER);
+    }
+    
+    uint64_t readRtcTime() 
+    {
+        while ( isRtcUpdating() );
+
+        uint8_t rtcRegisterStatusB = readRtcRegister(0x0B);
+        bool is24Hours = rtcRegisterStatusB & 0x02;
+        bool isBinary = rtcRegisterStatusB & 0x04;
+        Logf("Is 24 hours? %b - Is binary: %b - rtcRegisterStatusB: %d", LogSeverity::Warning , is24Hours, isBinary, rtcRegisterStatusB);
```

Comment:
do we want also to support both types? 12/24 hours?

Reply:
I think so, it's worth having the code, just in case some hardware uses those formats.